recipes/cutlass/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: cutlass
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cutlass/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1938f0e739646370a59ba1f5e365be4c701e8d9e0b9b725d306622e9aafa6b2a
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version in ("None", "10.2")]
+
```

```suggestion
  run_exports:
    - {{ pin_subpackage('cutlass', max_pin='x.x') }}

```
This package includes headers and a shared library, so it it meant to be linked against. Thus, it should have run_exports, so that it is automatically included as a dependency for packages that link to it. Please ensure that this version pinning is appropriate.

---

Thanks, I was considering this as well (but wanted to see if there's any other relevant feedback before retriggering CI).